.github/workflows/ci.yml

```diff
@@ -102,11 +106,10 @@ jobs:
 
   check-alembic-migrations:
     if: |
-      ${{ contains(github.event.pull_request.labels.*.name, 'require:db-migration')
-        && !contains(fromJSON('["flow:merge-queue", "flow:hotfix"]'), github.event.label.name)
-        && github.event.pull_request.merged == false
-        && needs.optimize-ci.outputs.skip == false
-      }}
+      contains(github.event.pull_request.labels.*.name, 'require:db-migration')
```

Comment:
Need to resolve the race issue with the auto-labeler workflow.
Let's embed the auto-labeler job in this workflow.

Reply:
This will be handled in a separate PR.